samples/opengl/cube/cube.cpp

```diff
@@ -34,6 +34,10 @@
     #include "../../sample.xpm"
 #endif
 
+#ifndef WGL_STEREO_ARB
+    #define WGL_STEREO_ARB  0x2012
+#endif
+
```

Comment:
Sorry, this is not going to work under non-MSW systems which don't use `WGL_STEREO_ARB` at all (but `GLX_STEREO` which has a completely different value).

Instead we need to change the logic in the sample and just pass `bool useStereo` to the ctor explicitly (or, better, use `int requestedFlags` with `Requested_Stereo` being the only defined flag for now).